packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -191,16 +192,14 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
         </Panel>
         {!isLoadingPeerProfile && isValidURL && (
           <Panel header={t('peerList.panels.information')}>
-            <Panel
-              bordered={true}
-              style={{
-                height: '200px',
-                marginBottom: '20px',
-                backgroundSize: 'cover',
-                backgroundImage: `url(${
-                  profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
-                }`
+            <ChooseEditImage
+              disabled
+              imageStyle={{
+                display: 'block',
+                marginLeft: 'auto',
+                marginRight: 'auto'
               }}
+              image={profile?.logo?.previewURL ?? '/static/placeholder-240x240.png'}
```

Comment:
is the `?? '/static/...'` necessary? you check for a valid url inside the component.

Reply:
I am sorry I don't get what you mean